server/handlebars/embed/public/style.css

```diff
@@ -43,54 +27,108 @@ h3#report-title {
 }
 
-#candidate-title > span {
-    font-family: Arial, Helvetica, sans-serif;
-    font-size: 12px;
-    font-weight: bold;
-    color: white;
-    background-color: #c25401;
-    padding: 8px 14px;
-    border-radius: 15px;
-    display: inline-block;
+.details > summary.recommended {
+  border: 1.5px solid #7ac498;
+  background-color: #e9fbe9;
+}
+.details > summary.recommended {
+  border: 1.5px solid #7ac498;
+  background-color: #e9fbe9;
+}
+
+div.recommended {
+  border-bottom: 1.5px solid #7ac498;
+  background-color: #e9fbe9;
 }
 
-#candidate-title.recommended {
-    border: 1.5px solid #7ac498;
-    background-color: #e9fbe9;
+.details > summary > ::before {
+    width: 32px;
+    height: 32px;
+    display: block;
+    content: '';
+    background-repeat: no-repeat;
+    position: absolute;
+    left: 5px;
+    top: -7px;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='30' height='31' viewBox='0 0 30 31'%3E%3Cg id='Group_1' data-name='Group 1' transform='translate(-11 -292)'%3E%3Cellipse id='Ellipse_1' data-name='Ellipse 1' cx='15' cy='15.5' rx='15' ry='15.5' transform='translate(11 292)' fill='%23d9ae4d'/%3E%3Cpath id='Path_3' data-name='Path 3' d='M4.524-17.368a4.507,4.507,0,0,0-1.56.312.278.278,0,0,0-.13.286V-14.9c0,2.184.416,8.294.468,9.022a.172.172,0,0,0,.182.156h2.08c.156,0,.182-.078.182-.156.078-.78.52-6.916.52-9.1v-1.872a.326.326,0,0,0-.156-.286A5.112,5.112,0,0,0,4.524-17.368Zm0,17.576A1.7,1.7,0,0,0,6.4-1.534,1.741,1.741,0,0,0,4.524-3.328,1.77,1.77,0,0,0,2.652-1.534,1.7,1.7,0,0,0,4.524.208Z' transform='translate(22 316)' fill='%23fff'/%3E%3C/g%3E%3C/svg%3E%0A");
 }
-#candidate-title.recommended > span {
-    background-color: #115b11;
+
+.details > summary.recommended ::before {
+  background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KICA8IS0tIUZvbnQgQXdlc29tZSBGcmVlIDYuNS4yIGJ5IEBmb250YXdlc29tZSAtIGh0dHBzOi8vZm9udGF3ZXNvbWUuY29tIExpY2Vuc2UgLSBodHRwczovL2ZvbnRhd2Vzb21lLmNvbS9saWNlbnNlL2ZyZWUgQ29weXJpZ2h0IDIwMjQgRm9udGljb25zLCBJbmMuLS0+CiAgPHBhdGgKICBmaWxsPSIjMTE1YjExIgogICAgZD0iTTI1NiA1MTJBMjU2IDI1NiAwIDEgMCAyNTYgMGEyNTYgMjU2IDAgMSAwIDAgNTEyek0yMTYgMzM2aDI0VjI3MkgyMTZjLTEzLjMgMC0yNC0xMC43LTI0LTI0czEwLjctMjQgMjQtMjRoNDhjMTMuMyAwIDI0IDEwLjcgMjQgMjR2ODhoOGMxMy4zIDAgMjQgMTAuNyAyNCAyNHMtMTAuNyAyNC0yNCAyNEgyMTZjLTEzLjMgMC0yNC0xMC43LTI0LTI0czEwLjctMjQgMjQtMjR6bTQwLTIwOGEzMiAzMiAwIDEgMSAwIDY0IDMyIDMyIDAgMSAxIDAtNjR6IgogIC8+Cjwvc3ZnPgo=');
 }
 
-#candidate-content-container {
-    border-left: 1.5px solid #c0c0c0;
-    border-right: 1.5px solid #c0c0c0;
-    border-bottom: 1.5px solid #c0c0c0;
+.details > summary > span {
+    position: relative;
+    padding-left: var(--left-right-padding);
+    padding-right: var(--left-right-padding);
+}
+
+.details > summary > span > h4 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1em;
+    color: #60470c;
+    display: inline-block;
+}
```

Comment:
It is a bit lost on me why an extra `<span>` is needed looking at the structure. But I also acknowledge that I'm looking at this after several revisions so perhaps this has already been explained.

Seems to me like the following should give the same result:

```css
.details > summary > h4 {
    position: relative;
    padding-left: var(--left-right-padding);
    padding-right: var(--left-right-padding);

    font-family: Arial, Helvetica, sans-serif;
    font-size: 1em;
    color: #60470c;
    display: inline-block;
}
```